### PR TITLE
Final tests

### PR DESCRIPTION
**Pull request by danieljsauer**
Add tests to season_manager and make alterations to reflect adjustments from this morning.

### CODE REVIEW

File: lib/league_manager.rb [hunk 1]
```diff
@@ -50,11 +51,11 @@ def lowest_scoring_home_team
  # -------Helper Methods-------
  
   def most_wins_team_id
-    @stat_tracker.average_goals_per_game.max_by { |id, goals| goals }[0]
+    @stat_tracker.goals_per_game_by_team_id.max_by { |id, goals| goals }[0]
```
Comment (garussell) [inline lib/league_manager.rb:54]: That was an easy fix!

File: lib/season_manager.rb [hunk 2]
```diff
@@ -15,8 +16,4 @@ def create_leagues(season_path)
       Season.new(seas)
     end
   end
-
-  def best_coach
```
Comment (garussell) [inline lib/season_manager.rb:19]: Thank you for removing these.

**Review by garussell — APPROVED** [review]
Looks good to me.  Nice job on the final push.

**Review by jpasko1112 — APPROVED** [review]
We can finally rest!

**Pull request merged**